pysat/tests/test_instrument.py

```diff
@@ -615,6 +615,57 @@ def test_getting_all_data_by_numpy_array_of_int(self):
         else:
             assert a.sizes['time'] == 5
 
+    # --------------------------------------------------------------------------
+    #
+    # Test variable renaming
```

Comment:
The first two tests I identify below can be combined into a single parametrized test.

Reply:
Thanks for the review @aburrell. Changes are in.